### PR TITLE
Add Universidade Federal de Uberlandia

### PR DESCRIPTION
**Pull request by klerisson**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [X] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: country-info.csv [hunk 1]
```diff
@@ -152,3 +152,4 @@ Zhejiang University,asia
 Bielefeld University,europe
 KTH Royal Institute of Technology,europe
 UFRGS,southamerica
+Universidade Federal de Uberlândia,southamerica
```
Comment (emeryberger) [inline country-info.csv]: For symmetry with Rio Grande do Sul, please abbreviate to UFU. Thanks.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: country-info.csv (hunk 1 above)
Comment (klerisson) [reply country-info.csv]: Thanks for the review.
The Brazilian universities: Universidade Federal de Minas Gerais, University of São Paulo, University of Campinas are not abbreviated. Just UFRGS is. Should I abbreviate to UFU?

Comment (klerisson) [reply country-info.csv]: Done.

**Review by emeryberger — APPROVED** [review]

**Comment by emeryberger** [comment]
Thanks!

File: country-info.csv (hunk 1 above)
Comment (emeryberger) [reply country-info.csv]: Some (but not all) of these names are really long and they mess up the UI. Thanks.

Comment (klerisson) [reply country-info.csv]: I'll be glad to abbreviate the rest.

Comment (emeryberger) [reply country-info.csv]: That'd be great, thanks.

Comment (emeryberger) [reply country-info.csv]: (separate PR)

Comment (klerisson) [reply country-info.csv]: Sure. Separate PR per university?

Comment (emeryberger) [reply country-info.csv]: One separate PR for all universities is fine. Thanks!

**Pull request merged**